client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.module.css

```
@@ -57,6 +57,14 @@
     /* … */
 }
 
+.results {
+  padding-left: 5px;
+  padding-top: 5px;
+  span {
+    font-weight: 600;
```

__Correction__
Use either of these weights, please:

```suggestion
    font-weight: 700;
```
```suggestion
    font-weight: bold;
```

__Reference__
- [Adobe Design Font Weight Screenshot](https://user-images.githubusercontent.com/62723358/119392296-2dd4c580-bc95-11eb-8039-2abd3e11d07e.png)
- [MDN: `font-weight`: Values](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#values)